cms/views/cms_views.py

```diff
@@ -242,13 +295,47 @@ def content(self):
 
         if page.language != self.locale:
             raise HTTPNotFound()
-        return {
+
+        context = {
             'page': page,
             'linked_pages': linked_pages,
             'primary_category': category,
             'content': markdown(page.content),
             'description': markdown(page.description),
         }
+        context.update(self.get_comment_context(page))
+        return context
+
+    @view_config(route_name='comments',
+                 renderer='cms:templates/comments/comment_page.pt')
```

Comment:
So we already have Jinja2 templates in this codebase and we'll be moving over to springboard _very soon_ so I'd rather not have any new templates in the old way (it just means more work later). And we won't be enabling commenting on the old templates (I refuse ;) )